snuba/migrations/operations.py

```diff
@@ -38,17 +38,17 @@ def format_sql(self) -> str:
 
 class RunSql(SqlOperation):
     def __init__(self, storage_set: StorageSetKey, statement: str) -> None:
-        self.statement = statement
         super().__init__(storage_set)
+        self.statement = statement
```

Comment:
is this just cosmetics (which. would be fine) or is there a logical reason why this had to be changed now ?

Reply:
No reason it needs to be changed as part of this, I just happened to notice it now. It shouldn't ever have been part of the public API of this class.

Reply:
Oops, sorry. You mean in reference to the order. Just cosmetics.